### PR TITLE
CONCF-505 | trending articles for main pages

### PR DESCRIPTION
**Pull request by Warkot**

### CODE REVIEW

File: front/scripts/main/app.ts [hunk 1]
```diff
@@ -76,7 +76,8 @@ App.initializer({
 	initialize: () => {
 		// 2870342045 -> Experiment ID in Optimizely / Mercury (production)
 		// 1 -> Variation ID that should have CuratedMainPages Enabled
-		M.prop('optimizelyCuratedMainPage', window.optimizely && optimizely.variationMap[2870342045] == 1);
+//		M.prop('optimizelyCuratedMainPage', window.optimizely && optimizely.variationMap[2870342045] == 1);
+		M.prop('optimizelyCuratedMainPage', true); // TODO: just for debugging - remove before merging
```
Comment (hakubo) [inline front/scripts/main/app.ts]: !!!

File: front/scripts/main/components/TrendingArticlesItemComponent.ts [hunk 2]
```diff
@@ -12,8 +12,8 @@ App.TrendingArticlesItemComponent = Em.Component.extend(App.ViewportMixin, {
 	emptyGif: 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAQAIBRAA7',
 	imageUrl: Em.computed.oneWay('emptyGif'),
 	href: Em.computed.oneWay('url'),
-	imageHeight: 150,
 	imageWidth: 250,
+	imageHeight: Em.computed(function () { return Math.floor(this.get('imageWidth') * 9 / 16); }),
```
Comment (hakubo) [inline front/scripts/main/components/TrendingArticlesItemComponent.ts]: () =>  Math.floor(this.get('imageWidth') * 9 / 16)

Comment (Warkot) [inline front/scripts/main/components/TrendingArticlesItemComponent.ts]: `(): number => Math.floor(this.get('imageWidth') * 9 / 16)`?

Comment (Warkot) [inline front/scripts/main/components/TrendingArticlesItemComponent.ts]: Something like this works;
```
imageHeight: Em.computed(function (): number => { return Math.floor(this.get('imageWidth') * 9 / 16); }),
```

Comment (hakubo) [inline front/scripts/main/components/TrendingArticlesItemComponent.ts]: so how about this:
```
imageHeight: Em.computed((): number => Math.floor(this.get('imageWidth') * 9 / 16))
```
if you omit {} in fat arrow function return is implicit

**Pull request closed without merging**